<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPeerDialog</class>
 <widget class="QWidget" name="TestPeerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Peer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter the peer details below.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Be careful! Do not blindly trust anyone that tells you to add their node.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="peerAddress">
       <property name="placeholderText">
        <string>Enter the peer's address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="peerPort">
       <property name="placeholderText">
        <string>Enter the peer's port</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Test!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
